go.mod

```diff
@@ -1,103 +1,104 @@
 module github.com/openshift/ptp-operator
 
-go 1.22
+go 1.22.0
```

Comment:
would 1.22 work?

Reply:
when I run go mod tidy it adds the `.0` so I cant undo it.